idearium-lib/lib/util/parse.js

```diff
@@ -16,4 +22,52 @@ const parseConfigAsBoolean = (val, defaultVal) => {
 
 };
 
-module.exports = { parseConfigAsBoolean };
+/**
+ * Parse a csv file.
+ * @example
+ * parseCsv(BufferData)
+ *     .then(data => console.log(data))
+ *     .catch(console.error);
+ * @param {Buffer|Stream} data Readable stream or buffered data contents.
+ * @param {Object} options Csv-parse options. http://csv.adaltas.com/parse/
+ * @param {Object} options.transform Optional transform function.
+ * @return {Promise} The parsed array contents.
+ */
+const parseCsv = (data, options) => new Promise((resolve, reject) => {
```

Comment:
Is there a capability to state the header does/doesn't have a header row and either parse or discard the first row accordingly?